docker/hq-compose.yml

```diff
@@ -117,13 +117,16 @@ services:
       - ${VOLUME_PREFIX}${BLANK_IF_TESTS-zookeeper:}/opt/zookeeper-3.7/data
 
   kafka:
-      image: bitnami/kafka:3.2
+      image: confluentinc/cp-kafka:7.2.2
```

Comment:
> The image release versioning is a bit odd, but can confirm that [v7.2.2 contains kafka version 3.2.2](https://docs.confluent.io/platform/current/release-notes/changelog.html#ak).

Can you note this here in a comment? Otherwise if I came across this line looking for our current kafka version I would be quite confused!

Reply:
Good point. See 5fae93be355ae6f5a56bfa4b58fef4273d9a429d